frontend/src/components/Roadmap/MilestoneCard.jsx

```diff
@@ -1,17 +1,18 @@
 import React, { useState } from 'react';
 import { ChevronDown, ChevronRight, Target, Calendar } from 'lucide-react';
 import TaskCard from './TaskCard';
+import { calculateMilestoneProgress } from '@/utils/roadmapUtils';
 
 /**
  * MilestoneCard - Displays a milestone with expandable functionality
  * 
  * Features:
  * - Shows milestone title and timeline
  * - Expandable/collapsible design with chevron indicators
- * - Progress bar visualization (placeholder - 0% progress)
- * - Task count display (placeholder - 0/0 tasks)
+ * - Progress bar visualization with real calculations
```

Comment:
nit: for the comment, there's no need to point out this is "real calculations". for real-world applications, we'd assume this is the real calculations. Something like "Progress bar visualization" will be sufficient here